### ISSUE TITLE
Expressly identify conditions

### ISSUE THREAD

**Issue by richardfontana** [copyleft-next/copyleft-next#23]
This is a suggestion from @pchestek:

> Given that US courts have to construe what is a “condition” and what is a “covenant,” although it's fairly obvious I would expressly identify what you consider to be the conditions. Paragraph 6 isn't a condition, so I excluded that from the enumeration, although ideally I would re-order the paragraphs and put 6 after current paragraph 10 so the enumeration would be easier. Alternatively, rather than listing the conditions by paragraph number, you could instead put the word “condition” in the title for all the relevant paragraphs.

**Comment by richardfontana** [comment]
Note that I followed @pchestek's suggestion to reorder sections in https://github.com/richardfontana/copyleft-next/pull/19#issuecomment-13220829. Separately, I identified the conditions in https://github.com/richardfontana/copyleft-next/commit/410102679a3130df808a5a6fb63cb7a3271fde18.

Although I kind of like the idea of instead putting the word 'condition' in the title of the relevant sections.
